### PR TITLE
Add Review Model And Relationship

### PR DESCRIPTION
**Pull request by brokenheart1712**

### CODE REVIEW

File: app/controllers/reviews_controller.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class ReviewsController < ApplicationController
+  belongs_to :user
```
Comment (vjnhat) [inline app/controllers/reviews_controller.rb]: ?????????????????????????

File: app/models/review.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class Review < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :book
+  validates :user_id, presence: true
```
Comment (NguyenTanDuc) [inline app/models/review.rb]: Add new line

File: app/models/user.rb [hunk 3]
```diff
@@ -1,4 +1,5 @@
 class User < ActiveRecord::Base
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
+  has_many :reviews, dependent: :destroy
```
Comment (NguyenTanDuc) [inline app/models/user.rb]: new line

**Comment by NguyenTanDuc** [comment]
ok

File: app/models/book.rb [hunk 4]
```diff
@@ -0,0 +1,4 @@
+class Book < ActiveRecord::Base
+  has_many :reviews, dependent: :destroy
+
```
Comment (khanhkhanh) [inline app/models/book.rb]: remove this line

File: app/models/review.rb [hunk 5]
```diff
@@ -0,0 +1,10 @@
+class Review < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :book
+  
+  validates :user_id, presence: true
+  validates :book_id, presence: true
+  validates :content, presence: true
+  validates :rating, presence: true
+
```
Comment (khanhkhanh) [inline app/models/review.rb]: Remove again

File: app/models/user.rb [hunk 6]
```diff
@@ -1,4 +1,7 @@
 class User < ActiveRecord::Base
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
+         
+  has_many :reviews, dependent: :destroy
+
```
Comment (khanhkhanh) [inline app/models/user.rb]: Again

**Comment by khanhkhanh** [comment]
Okay

**Pull request merged**